src/plugins/home/common/instruction_variant.ts

```diff
@@ -52,6 +53,12 @@ const DISPLAY_MAP = {
     defaultMessage: 'Elastic APM in Fleet',
   }),
   [INSTRUCTION_VARIANT.OPEN_TELEMETRY]: 'OpenTelemetry',
+  [INSTRUCTION_VARIANT.OTHER_LINUX]: i18n.translate(
+    'home.tutorial.instruction_variant.other_linux',
```

Comment:
@Bamieh is there anything special we need to do in order for the new _key_ to be correctly taken into account / translated?

Reply:
I updated the i18n here https://github.com/elastic/kibana/pull/162977/commits/fba082baa4da86f85ecb32b285e4b577d6ecec3d